TAF/utils/scripts/docker/sync-nightly-build.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+# TODO: Change URl and file spec to use edgexfoundry master once developer-scripts PR is merged
+NIGHT_BUILD_URL="https://codeload.github.com/lenny-intel/developer-scripts/zip/multi2"
+FILE_SPEC="developer-scripts-multi2/releases/nightly-build/compose-files/*"
+
+# x86_64 or arm64 for make run option
+[ "$(uname -m)" != "x86_64" ] && USE_ARM64="arm64"
+
+# security or no security for make run option
+[ "$SECURITY_SERVICE_NEEDED" != true ] && USE_NO_SECURITY="no-secty"
+
+IS_ALPINE=`uname -a | grep linuxkit`
+# make sure curl is installed if running from apline image
+if [ "$IS_ALPINE" != ""] ; then
+  apk add curl
```

Comment:
@lenny-intel Is curl required for this script? If yes, I can add it to the compose image.

Reply:
I switched to using wget, but curl works better on windows bash. So yes... :-)